static/src/javascripts/projects/commercial/modules/hosted/onward.js

```diff
@@ -0,0 +1,54 @@
+// @flow
+import config from 'lib/config';
+import mediator from 'lib/mediator';
+import fetchJson from 'lib/fetch-json';
+import fastdom from 'lib/fastdom-promise';
+import HostedCarousel from 'commercial/modules/hosted/onward-journey-carousel';
+
+const loadOnwardComponent = (
+    start: () => void,
+    stop: () => void
+): Promise<any> => {
+    start();
+
+    const placeholders = document.getElementsByClassName(
+        'js-onward-placeholder'
```

Comment:
So I have a business question... will we ever have more than one placeholder on a page?

I would like to simplify the hosted-onward stuff so some more comprehensive testing can be added, working with Flow types of `HTMLCollection<HTMLElement>` is painful.

If we only ever return a collection of length 0 or 1, that also means I can nuke the for loop further down...

Anyone know?